superset/examples/helpers.py

```diff
@@ -23,16 +23,17 @@
 from urllib import request
 
 from superset import app, db
-from superset.connectors.connector_registry import ConnectorRegistry
+from superset.datasource.dao import DatasourceDAO
 from superset.models.slice import Slice
+from superset.utils.core import DatasourceType
 
 BASE_URL = "https://github.com/apache-superset/examples-data/blob/master/"
 
 misc_dash_slices: Set[str] = set()  # slices assembled in a 'Misc Chart' dashboard
 
 
 def get_table_connector_registry() -> Any:
-    return ConnectorRegistry.sources["table"]
+    return DatasourceDAO.sources[DatasourceType.TABLE]
```

Comment:
can we use sqlatable lookup directly instead of the DatasourceDAO since it's only one type?